misaligned/misaligned.java

```diff
@@ -3,16 +3,18 @@ static int printColorMap() {
         String majorColors[] = {"White", "Red", "Black", "Yellow", "Violet"};
         String minorColors[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
         int i = 0, j = 0;
-        for(i = 0; i < 5; i++) {
-            for(j = 0; j < 5; j++) {
-                System.out.printf("%d | %s | %s\n", i * 5 + j, majorColors[i], minorColors[i]);
+         int count = 0;
+        for(i = 0; i < majorColors.length; i++) {
+            for(j = 0; j < minorColors.length; j++) {
+                System.out.printf("%d | %s | %s \n", i * minorColors.length + j, majorColors[i], minorColors[j]);
+                count++;
             }
         }
-        return i * j;
+        return count-1;

```

Comment:
This introduces a extra bug in the production code to make the test to fail. Instead, change the test code to catch the already existing bug in the pair-number calculation.
Hint: Separate the pair-number calculation into a function of its own, and test that one.